Formula/wine.rb

```diff
@@ -34,8 +34,8 @@ class Wine < Formula
     url "https://source.winehq.org/git/wine.git"
 
     resource "mono" do
-      url "https://dl.winehq.org/wine/wine-mono/4.7.3/wine-mono-4.7.3.msi"
-      sha256 "d24a8017371c7e8224a1778bb43a113ed7ed9720efd9d0cda175d42db6106d3a"
+      url "https://dl.winehq.org/wine/wine-mono/4.7.5/wine-mono-4.7.5.msi"
```

Comment:
Since these seem to be the same, could you move them out of the `head do` block?